_includes/header.html

```diff
@@ -17,7 +17,7 @@
   <div class="syn-page-header-content syn-flex justify-space-between">
     <div class="syn-flex align-center">
       <a class="syn-flex" href="/">
-        <img class="syn-ml-8" src="/v1/config/logo" style="max-height: '24px'; max-width: '100%';" />
+        <img class="syn-ml-8" src="/v1/config/logo" style="max-height: 40px; max-width: 100%;" />
```

Comment:
👍 This thumbs-up is for getting rid of the weird quotes.

Reply:
🪓